### DISCUSSION TITLE
Which application do you use to make icons?

### DISCUSSION THREAD

**Discussion by vapormermaid** [nestoris/Win98SE#26] [General]
I am a big fan of this icon theme and I would like to try and contribute some icons, but I'm not sure what pixel art program I should use (I haven't regularly worked on pixel art for about a decade). Do you have any recommendations?

**Comment by nestoris** [comment]
Please write to me via [Telegram](https://t.me/dobromils). It seems that soon GitHub will close its access to the Russians (2-factor auth + Russian cells not allowed), and the project will move to another hosting.

**Comment by Dejweed** [comment]
Check out GIMP and IconLover.

**Reply by nestoris** [reply]
Krita also has a palletify filter... It does not make such perfect diffusions as Photoshop (for example as in folder icon), but very useful for realtime RGB editing with 256-color indexed preview.
*I used a gradient transparent mask to compare RGB gradient and indexed color with pattern blending:*
![krita-palletify](https://github.com/nestoris/Win98SE/assets/26491673/4cdc4204-d979-480d-b401-820381072e68)
